### DISCUSSION TITLE
GitHub Contributions Not Showing Despite Correct Username and Email Configuration

### DISCUSSION THREAD

**Discussion by prmayanksingh** [community/community#136954] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I am facing an issue where my contributions are not appearing on my GitHub profile. I have ensured that I have set the correct username and email in my local Git configuration, and the email is also verified on my GitHub account. Despite this, my contributions are still not showing up.

I have tried the 'Resync contributions' option and confirmed that the commits are in the main branch of a public repository. However, the problem persists.

Could someone please help me resolve this issue?

Thank you in advance for your assistance.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @prmayanksingh , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category.  I've gone ahead and moved it for you. Good luck!

**Comment by AyhemKouki** [comment]
Check Commit Attribution: Even with the correct username and email, sometimes commits may not be attributed if they’re authored by another account or if there are discrepancies in capitalization. Make sure the email in your commits exactly matches the primary email on your GitHub profile.

Verify Repository Visibility: Since contributions count only if they’re in public repositories, double-check that the repository visibility hasn’t been changed back to private after pushing your commits. You can confirm this in the repository’s settings under Repository visibility.

Confirm the Main Branch: GitHub only tracks contributions on the default branch (often main or master). If your commits are on another branch, consider merging them into the default branch if appropriate, or switching the repository’s default branch.

Double-Check Date and Time on Commits: GitHub uses timestamps to log contributions on specific days. If your local machine's date or time was incorrect when you made the commits, GitHub might not recognize them as occurring on the correct dates. If you identify any errors in the timestamps, you can use git rebase to amend commit dates.

Try a Hard Refresh on Your Profile: Sometimes, GitHub profiles don’t update in real-time. A hard refresh (Ctrl+F5 on most browsers) might force the contributions to display.
